tests/test_advantage.py

```diff
@@ -2,7 +2,7 @@
 
 from requests.exceptions import HTTPError
 
-from webapp.advantage.api import (
+from webapp.advantage.ua_contracts.api import (
```

Comment:
Maybe you don't actually need the word "api" here - maybe it could just be `ua_contracts`? Or whatever the best way to describe their system is.

Reply:
It is. The name of the folder is just `ua_contracts`. Inside it we have a file called `api`.